presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java

```diff
@@ -1548,7 +1671,7 @@ public void testMaterializedViewForJoin()
 
             assertUpdate(format("REFRESH MATERIALIZED VIEW %s WHERE ds='2020-01-01'", view), 65025);
 
-            String viewQuery = format("SELECT view_orderkey, view_totalprice from %s where view_orderkey <  10000", view);
+            String viewQuery = format("SELECT view_orderkey, view_totalprice from %s where view_orderkey < 10000", view);
```

Comment:
I think we can add the assert plan for this test case as well. So once it is enabled we have the assert plan ready.

Reply:
will do after the depending PR gets merged so I can look into the actual query plan and make sure this test isn't broken.